app/components/event_banner.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+//need to load next event data and have banner link to single event page (slug)
+
+export default function EventBanner() {
+  const banner_style = {
+    color: "white",
+    backgroundColor: "#A379C9",
+    padding: "10px",
+    textAlign: "center",
```

Comment:
Lets look into styled components or use Chakra Box. https://chakra-ui.com/docs/components/box

If you install Chakra then we can start wrapping things up in a theme https://chakra-ui.com/docs/styled-system/customize-theme

This will give us branding colors to set in a global rather than copy pasta in single components.